tests/e2e/demo-homepages.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { isTesting } from '@lmc-eu/spirit-common/constants/environments';
 import { SERVERS, getDevelopmentEndpointUri } from '@lmc-eu/spirit-common/constants/servers';
 import { test } from '@playwright/test';
-import { takeScreenshot, waitForPageLoad } from '../helpers';
+import { hideFromVisualTests, takeScreenshot, waitForPageLoad } from '../helpers';
```

Comment:
it hides something? 🤔

Reply:
It's bound to a CSS helper, I cannot tell what exactly it hides…